### PR TITLE
Manual Edits - Provided by Gojo

### PR DESCRIPTION
**Pull request by MiteyDrill**
Gojo provided a lot of info from a large google document and I spent 
some time formatting it for html. This is just initial commit for users 
to have, still need to go over it and make it look better

### CODE REVIEW

**Comment by MiteyDrill** [comment]
image
Manual looks like this now, hard to see but its a lot of info

File: manual.php [hunk 1]
```diff
@@ -19,119 +19,160 @@
...
+
+          <h2>Bloodline</h2>
+          <p>Bloodline rates for your first roll upon reaching Genin are as followed</p>
+          <p><b>None</b> - 50% chance</p>
+          <p><b>Lesser</b> - 20% chance</p>
+          <p><b>Common</b> - 15% chance</p>
+          <p><b>Elite</b> - 10% chance</p>
+          <p><b>Legendary</b> - 5% chance</p>
+
+          <p>Use Ancient Kunai to purchase or reroll to a new bloodline of your choosing. Note, 50% of bloodline skill will be lost when acquiring a new bloodline.</p>
```
Comment (levimeahan) [inline manual.php:127]:
```suggestion
          <p>Use Ancient Kunai to purchase or reroll to a new bloodline of your choosing. Note, 10% of bloodline skill will be lost when acquiring a new bloodline.</p>
```

File: manual.php [hunk 2]
```diff
@@ -19,119 +19,160 @@
...
+          <h2>Character Rank, Stat caps and Requirements</h2>
+          <p><b>Academy student</b>- Starting rank up to level 10. Stat cap of 500</p>
+          <p><b>Genin</b> - Must be a lvl 10 AS plus required exp to take the exam. Stat cap of 3000</p>
 
           <p>
-            Add profile info here
+            In order to pass the exam, you must buy and learn <b>Basic Replacement</b>, <b>Clone Combo</b>, and <b>Transformation Trickery</b> from the shop and perform the hand signs.<br>
+          </p>
+          <p>
+            <b>Chuunin</b> - Must be a lvl 20 Genin plus required exp to take the exam. Stat cap of 15000
```
Comment (levimeahan) [inline manual.php:92]:
```suggestion
            <b>Chuunin</b> - Must be a lvl 20 Genin plus required exp to take the exam. Stat cap of 25000
```

File: manual.php [hunk 3]
```diff
@@ -19,119 +19,160 @@
...
-          <!--User Profile-->
-          <div id='manual_profile_wrapper'>
-            <div id='manual_profile_header'>
-              <h2 style='display: inline'>User Profile</h2> <!--This is only a for an example, do not style inline unless absolutely necessary-->
-              <img width='40%' src="./images/manual/navigation_profile.png"/>
-              <h3>Navigating the Profile Page</h3>
-            </div>
+          <h2>Character Rank, Stat caps and Requirements</h2>
+          <p><b>Academy student</b>- Starting rank up to level 10. Stat cap of 500</p>
+          <p><b>Genin</b> - Must be a lvl 10 AS plus required exp to take the exam. Stat cap of 3000</p>
```
Comment (levimeahan) [inline manual.php:86]:
```suggestion
          <p><b>Genin</b> - Must be a lvl 10 AS plus required exp to take the exam. Stat cap of 5000</p>
```

File: manual.php [hunk 4]
```diff
@@ -19,119 +19,160 @@
...
 
-          <!--User Profile-->
-          <div id='manual_profile_wrapper'>
-            <div id='manual_profile_header'>
-              <h2 style='display: inline'>User Profile</h2> <!--This is only a for an example, do not style inline unless absolutely necessary-->
-              <img width='40%' src="./images/manual/navigation_profile.png"/>
-              <h3>Navigating the Profile Page</h3>
-            </div>
+          <h2>Character Rank, Stat caps and Requirements</h2>
+          <p><b>Academy student</b>- Starting rank up to level 10. Stat cap of 500</p>
```
Comment (levimeahan) [inline manual.php:85]:
```suggestion
          <p><b>Academy student</b>- Starting rank up to level 10. Stat cap of 1000</p>
```

**Review by levimeahan — APPROVED** [review]

**Comment by levimeahan** [comment]
image
One follow up to fix

**Pull request merged**